Analysis/Tasks/PWGHF/HFCandidateCreator2Prong.cxx

```diff
@@ -179,6 +181,228 @@ struct HFCandidateCreator2ProngMC {
+DECLARE_SOA_COLUMN(NSigTOF_Ka_1, nsigTOF_Ka_1, float);
+DECLARE_SOA_COLUMN(DecayLength, decayLength, float);
+DECLARE_SOA_COLUMN(DecayLengthXY, decayLengthXY, float);
+DECLARE_SOA_COLUMN(DecayLengthNormalised, decayLengthNormalised, float);
+DECLARE_SOA_COLUMN(DecayLengthXYNormalised, decayLengthXYNormalised, float);
+DECLARE_SOA_COLUMN(CPA, cpa, float);
+DECLARE_SOA_COLUMN(CPAXY, cpaXY, float);
+DECLARE_SOA_COLUMN(Ct, ct, float);
+DECLARE_SOA_COLUMN(ImpactParameterProduct, impactParameterProduct, float);
+DECLARE_SOA_COLUMN(CosThetaStar, cosThetaStar, float);
+DECLARE_SOA_COLUMN(MCflag, mcflag, uint8_t);
+// Events
+DECLARE_SOA_COLUMN(IsEventReject, isEventReject, int);
+DECLARE_SOA_COLUMN(RunNumber, runNumber, int);
+} // namespace full
+
+DECLARE_SOA_TABLE(HfCandProng2Full, "AOD", "HFCANDP2Full",
+                  collision::BCId,
+                  collision::NumContrib,
+                  collision::PosX,
+                  collision::PosY,
+                  collision::PosZ,
+                  hf_cand::XSecondaryVertex,
+                  hf_cand::YSecondaryVertex,
+                  hf_cand::ZSecondaryVertex,
+                  hf_cand::ErrorDecayLength,
+                  hf_cand::ErrorDecayLengthXY,
+                  hf_cand::Chi2PCA,
+                  full::RSecondaryVertex,
+                  full::DecayLength,
+                  full::DecayLengthXY,
+                  full::DecayLengthNormalised,
+                  full::DecayLengthXYNormalised,
+                  full::ImpactParameterNormalised0,
+                  full::PtProng0,
+                  full::PProng0,
+                  full::ImpactParameterNormalised1,
+                  full::PtProng1,
+                  full::PProng1,
+                  hf_cand::PxProng0,
+                  hf_cand::PyProng0,
+                  hf_cand::PzProng0,
+                  hf_cand::PxProng1,
+                  hf_cand::PyProng1,
+                  hf_cand::PzProng1,
+                  hf_cand::ImpactParameter0,
+                  hf_cand::ImpactParameter1,
+                  hf_cand::ErrorImpactParameter0,
+                  hf_cand::ErrorImpactParameter1,
+                  full::NSigTPC_Pi_0,
+                  full::NSigTPC_Ka_0,
+                  full::NSigTOF_Pi_0,
+                  full::NSigTOF_Ka_0,
+                  full::NSigTPC_Pi_1,
+                  full::NSigTPC_Ka_1,
+                  full::NSigTOF_Pi_1,
+                  full::NSigTOF_Ka_1,
+                  full::M,
+                  full::ImpactParameterProduct,
+                  full::CosThetaStar,
+                  full::Pt,
+                  full::P,
+                  full::CPA,
+                  full::CPAXY,
+                  full::Ct,
+                  full::Eta,
+                  full::Phi,
+                  full::Y,
+                  full::E,
+                  full::MCflag);
+
+DECLARE_SOA_TABLE(HfCandProng2FullEvents, "AOD", "HFCANDP2FullE",
```

Comment:
I think that table descriptors are supposed to be in upper case letters only.
Maybe @jgrosseo or @aalkin can comment.

Reply:
That is not necessary, the only strict requirement is for descriptor to be within 16 characters.